src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,68 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String SUPPLY_WORD = "supply,";
+    private static final String BUY_WORD = "buy,";
+    private static final String RESULT_WORD = "result,";
+    private String result = "";
```

Comment:
I think it's a bad idea to make result as a class level variable. Just create new one in getResult() method
```suggestion
```